src/service/extensions-impl.js

```diff
@@ -445,13 +446,18 @@ export class Extensions {
     stubLegacyElements(childWin);
 
     const promises = [];
-    extensionIds.forEach(extensionId => {
+    extensionData.forEach(extension => {
+      let extensionId;
+      if (extension && typeof extension == 'object') {
```

Comment:
A4A should be passing down the `custom-element` fields directly. There's no reason for `extension-impl.js` to know anything about A4A objects.